gtt/gtt_display.py

```diff
@@ -182,3 +182,411 @@ def update_bar_value(self, bar_id: IdType, value: int):
         )
 
         self._receive_status_response(252, 105)
+
+    def create_pixel(self, x_pos: int, y_pos: int):
+        """Creates a bar graph which is really just a single bar.
+        :param x_pos: the distance from the left edge of the screen in pixel
+        :param y_pos: the distance from the top edge of the screen in pixel
+        """
+
+        self._validate_x(x_pos)
+        self._validate_y(y_pos)
+        self._conn.write(
+            bytes.fromhex('FE 70') +
+            ints_to_signed_shorts(x_pos, y_pos)
+        )
+
+    def clear_all_traces(self):
+        self._conn.write(
+            bytes.fromhex('FE 77')
+        )
+
+    def draw_rectangle(self, x_pos: int, y_pos: int, width: int, height: int):
+        """Creates a rectangle which is really just a single bar.
+        :param x_pos: the distance from the left edge of the screen in pixels
+        :param y_pos: the distance from the top edge of the screen in pixels
+        :param width: the width of the rectangle in pixels
+        :param height: the height of the bar in pixels
+        """
+
+        self._validate_x(x_pos, x_pos + width - 1)
+        self._validate_y(y_pos, y_pos + height - 1)
+        self._conn.write(
+            bytes.fromhex('FE 72') +
+            ints_to_signed_shorts(x_pos, y_pos, width, height)
+        )
+
+    def create_a_label(self, label_id: int, x_pos: int, y_pos: int, width: int, height: int, font: int, R: int, G: int,
```

Comment:
- probably change name to `create_label`
- Combine R, G, and B into a single hex string as I did in create_bar
- change `size` to `font_size`
- This is up for debate, but I don't think this needs to have a `value` parameter. There's a default value on the bar because the bar won't show up if it doesn't have a value, but in the case of a label I wouldn't expect it to show up without updating the value. Minimally, `value` should have a default value like `value='Label'`
- rot should have an enum and a docstring entry
- `label_id` should be an `IdType` not an `int`